src/components/Main/Articles/Articles.jsx

```diff
@@ -6,29 +6,72 @@ import './Article.css';
 import Loading from '../../UI/Loading';
 import Error from '../../UI/error';
 import Topics from './Topics';
-
+import { useSearchParams } from 'react-router-dom';
 
 const Articles = () => {
-
   const [articles, setArticles] = useState([]);
   const [error, setError] = useState(null);
+  const [sortCriteria, setSortCriteria] = useState('created_at');
+  const [sortOrder, setSortOrder] = useState('desc');
+  const [searchParams, setSearchParams] = useSearchParams();
 
   useEffect(() => {
-  
-    axios.get('https://nc-news-vvdv.onrender.com/api/articles')
-      .then(response => setArticles(response.data.articles))
+    const sort_by = searchParams.get('sort_by') || 'created_at';
+    const order = searchParams.get('order') || 'desc';
+
+    axios.get('https://nc-news-vvdv.onrender.com/api/articles', {
```

Comment:
It would be nice if this axios function was in a separate file (api.js) to keep our code more modular